operator/bundle_helpers/patch-csv.py

```diff
@@ -80,13 +80,15 @@ def patch_csv(csv_doc, version, operator_image, first_version, no_related_images
     if replaced_xyz is not None:
         csv_doc["spec"]["replaces"] = f"{raw_name}.v{replaced_xyz}"
 
-    if no_related_images:
-        if 'relatedImages' in csv_doc['spec']:
-            # OSBS fills relatedImages therefore we must not provide that ourselves.
-            # Ref https://osbs.readthedocs.io/en/latest/users.html?highlight=relatedImages#creating-the-relatedimages-section
-            del csv_doc['spec']['relatedImages']
-    else:
-        csv_doc['spec']['relatedImages'] = construct_related_images()
+    related_images = construct_related_images()
+    if related_images:
+        # If related images are specified in the environment, inject them into the CSV.
+        csv_doc['spec']['relatedImages'] = related_images
```

Comment:
This will execute downstream right? We don't want that because OSBS will take care of `spec.relatedImages`.

Reply:
discussed offline.